File: C++/program4.cpp
#include <iostream>
using namespace std;

class Percentage 
{
public:
    float calculatePercentage(int iObtain, int iTotal) 
    {
        float fPercentage = ((float)iObtain / (float)iTotal) * 100;
        return fPercentage;
    }
};

int main() {
    int iValue1 = 0, iValue2 = 0;
    float fRet = 0.0f;

    cout << "Enter the Obtain Marks: ";
    cin >> iValue1;

    cout << "Enter the Total Marks: ";
    cin >> iValue2;

    Percentage pobj;
    fRet = pobj.calculatePercentage(iValue1, iValue2);

    cout << "Percentage is: " << fRet<<"%";

    return 0;
}

File: JAVA/program4.java
import java.util.*;
class Percentage
{
    public float Percentage(int iOptain, int iTotal)
    {
        float fPercentage = 0.0f;
        fPercentage = ((float)iOptain / (float)iTotal) * 100;
        return fPercentage;
    }
}
class program4
{
    public static void main(String[] A)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;
        float fRet = 0.0f;

        System.out.println("Enter the Optain Marks :");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the Total Marks :");
        iValue2 = sobj.nextInt();

        Percentage pobj = new Percentage();
        fRet = pobj.Percentage(iValue1, iValue2);

        System.out.println("Percentage is:"+fRet);
    }
}